body.go

```diff
@@ -9,4 +18,15 @@ type Body struct {
 	File       interface{} `json:"file,omitempty"`
 	GraphQL    interface{} `json:"graphql,omitempty"`
 	Disabled   bool        `json:"disabled,omitempty"`
+	Options    BodyOptions `json:"options,omitempty"`
```

Comment:
Here `BodyOptions` must be a pointer as we want it to be nullable, otherwise it will always have a value.

This is why the `TestRequestMarshalJSON` is failing.

Reply:
done.